SillySCP/API/IInittable.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SillySCP.API
+{
+    public interface IInittable
```

Comment:
IRegisterable?

Reply:
probably a better name yea, i put a temp one as an example
its a draft PR

Reply:
Changed to IRegisterable @moddedmcplayer